source/index.html

```diff
@@ -1,12 +1,149 @@
+      <img class="hero__image" src="img/hero-image.png" alt="Функциональное питание для котов">
+    </section>
+
+    <section class="food-type">
+
+      <article class="food-type__item">
+        <img src="img/cat-slim.svg" alt="Похудение" width="100" height="106">
+        <h2 class="food-type__title">Похудение</h2>
+        <p class="food-type__text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на
+          диету.</p>
+        <a class="food-type__link" href="#">каталог slim</a>
+      </article>
+
+      <article class="food-type__item">
+        <img src="img/cat-mass.svg" alt="Набор массы" width="134" height="100">
+        <h2 class="food-type__title">Набор массы</h2>
+        <p class="food-type__text">Заработать авторитет среди дворовых котов и даже собак? Набор массы поможет</p>
+        <a class="food-type__link" href="#">каталог pro</a>
+      </article>
+    </section>
+
+    <section class="how-it-works">
+      <h2 class="how-it-works__title">Как это работает</h2>
+      <ul class="how-it-works__list">
+        <li class="how-it-works__item">
+          <p class="how-it-works__text">
+            Функциональное питание содержит только полезные питательные вещества.
+          </p>
+        </li>
+        <li>
+          <p class="how-it-works__text">
+            Выпускается в виде порошка, который нужно лишь залить кипятком и готов
+          </p>
+        </li>
+        <li class="how-it-works__item">
+          <p class="how-it-works__text">
+            Замените один-два приема обычной еды на наше функциональное питание.
+          </p>
+        </li>
+        <li class="how-it-works__item">
+          <p class="how-it-works__text">
+            Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+          </p>
+      </ul>
+    </section>
+
+    <section class="example">
+      <h2 class="example__title">Живой пример</h2>
+      <p class="example__text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim.
+        Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов
+        в день. </p>
+      <dl class="example__info-list">
+        <div class="example__info-item">
+          <dt class="example__info-title">5 кг</dt>
+          <dd class="example__info-text">снижение веса</dd>
+        </div>
+        <div class="example__info-item">
+          <dt class="example__info-title">60 дней</dt>
+          <dd class="example__info-text">затрачено времени</dd>
+        </div>
+      </dl>
+      <div class="example__cost-wrapper">
+        <span class="example__cost-title"> Затраты на питание:</span>
+        <span class="example__cost">15 000 руб.</span>
+      </div>
+    </section>
+    <div class="example-slider">
+      <img class="example__image" src="img/cat-example-before.png" alt="Пример до" width="560" height="512">
```

Comment:
```suggestion
      <img class="example-slider__image" src="img/cat-example-before.png" alt="Пример до" width="560" height="512">
```
Картинка - это элемент блока `example-slider`
Также [Йоксель](https://yoksel.github.io/html-tree/) нам ругается на картинки
![image](https://github.com/htmlacademy-adaptive/1919993-cat-energy-1/assets/30808818/3be5c991-9f1f-42f7-ad93-3dcd2c992b61)

С картинкой ниже тоже нужно поправить название класса